build-coordinator/src/test/java/org/jboss/pnc/core/test/mock/BuildDriverMock.java

```diff
@@ -40,7 +40,7 @@
 
     @Override
     public String getDriverId() {
-        return null;
+        return "termd-build-driver";
```

Comment:
It will complicate things, we need to use mock also in configs and real name for the tests that don't use a mocked one.